.github/workflows/pull_request.yml

```diff
@@ -21,8 +21,8 @@ jobs:
             ~/.cargo/git
             ~/.cargo/registry
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
-      - uses: boa-dev/criterion-compare-action@master
+      - uses: boa-dev/criterion-compare-action@v3.0.1
```

Comment:
Is there a reason for pinning the action to the version 3.0.1? We could use `main` to avoid having to bump the version on every new change.

Reply:
The idea behind this is to follow the nomenclature with the rest of actions. This action is now being used by more projects, not only Boa, so we should use the released versions. That way we can also develop the action in the `main` branch as needed, and release new versions with new features. Also, @dependabot will create PRs to update the action if we do it like this.